### ISSUE TITLE
Zero padding in hexlify

### ISSUE THREAD

**Issue by joesfer** [michalbe/binascii#11]
Hi, I've noticed that the hex encoding produced by the hexlify function was not entirely matching the result generated by Python, after some digging the difference was in the lack of 0 padding for those digits < 10. I've corrected this locally as follows:
```javascript
var hexlify = function(str) {
	var result = '';
	var padding = '00';
	for (var i=0, l=str.length; i<l; i++) {
		digit = str.charCodeAt(i).toString(16);
		padded = (padding+digit).slice(-2);
		result += padded;
	}
	return result;
};
```

Which seems to do the trick, however I haven't run your unit tests to verify 100%. Hope it helps, though. Thanks for sharing binascii.

**Comment by michalbe** [comment]
Hey @joesfer, sorry for the late reply. Thanks for great catch! Do you want to prepare the patch and send me a PR? I'm not sure if I'll find spare time in the near future to fix it myself.

**Comment by michalbe** [comment]
Closed by #12
